activefires_pp/post_processing.py

```diff
@@ -101,6 +101,10 @@ def __init__(self, filepath=None, afdata=None, platform_name=None, timezone='GMT
         self.timezone = timezone
         self.platform_name = platform_name
 
+    def set_af_data(self, afdata):
+        """Set the active fires data frame."""
+        self._afdata = afdata
```

Comment:
Why not make the `afdata` a public attribute if it can be changed directly anyway?